docs/packages.md

```diff
@@ -24,6 +24,22 @@ The Linux Software Repository can be configured to automatically install the Lin
 > Optionally, if you prefer manual configuration, the Linux Software Repository configuration files are available at [https://packages.microsoft.com/config](https://packages.microsoft.com/config/). The name and location of these files can be located using the following URI naming convention:
 `https://packages.microsoft.com/config/<Distribution>/<Version>/prod.(repo|list)`
 
+### Directory structure
+
+At the root of packages.microsoft.com, you’ll see folders like:
+
+- `/config/<distro>/<version>/prod.list`: Contains repo configuration files for each supported distribution.
+- `/repos/`: Contains product-specific repositories (for example, edge, azure-cli, microsoft-prod).
+- `/ubuntu/`: Contains distribution-specific repos for Microsoft packages that integrate with Ubuntu’s APT system.
+
+A few things to keep in mind regarding the packages.microsoft.com repo structure:
+
+- The packages.microsoft.com repository is public and designed for programmatic consumption by package managers, not for manual browsing, though directory listing is enabled for convenience.
+- The repos tree is product-centric, not distribution-centric. For example: `repos/edge/`, `repos/azure-cli/`, repos/microsoft-prod/', etc. Each contains metadata for multiple distributions, so the same product repo can serve Ubuntu, Debian, etc.
```

Comment:
- Some repos are product-centric, not distribution-centric. ... See individual product documentation for the preferred installation instructions.